src/modules/Builder/pages/BuilderApplet/BuilderApplet.tsx

```diff
@@ -61,17 +64,21 @@ export const BuilderApplet = () => {
   } = methods;
 
   useEffect(() => {
-    if (loadingStatus === 'success' && !isNewApplet) {
+    if (location.state?.isFromLibrary) setIsFromLibrary(true);
+  }, [location.state]);
+
+  useEffect(() => {
+    if (
+      loadingStatus === 'success' &&
```

Comment:
please change type `MetaStatus` with enum and use its value

Reply:
it will cause a lot of changes and typo fixes (around 40)